crates/fluvio-sc-schema/src/objects/classic.rs

```diff
@@ -412,6 +413,197 @@ mod delete_macro {
     pub(crate) use ClassicDecodingDelete;
 }
 
+/// carry from prev version for compatibility test
+mod update_macro {
+
```

Comment:
should not be needed.  only for supporting old protocol

Reply:
Make sense! We don't need this. 

I removed that.